src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -19,11 +20,11 @@ public Movie add(Movie movie) {
 
     @Override
     public Movie get(Long id) {
-        return movieDao.get(id).get();
+        return movieDao.get(id).orElseThrow(() -> new DataProcessingException("No such movie"));
```

Comment:
According to the checklist, it's recommended to return `Optional` in `get()` method on the DAO layer, and the actual entity on the service layer. Here, you're correctly returning the entity on the service layer, but make sure the DAO layer also follows this convention.